### PR TITLE
[REF] Make use of recently added default pager size setting in Reports

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This makes use of the recently added default_pager_size setting in reports

Before
----------------------------------------
Constant used, no reference to setting

After
----------------------------------------
Setting used

ping @demeritcowboy @colemanw

### CODE REVIEW

File: CRM/Report/Form.php [hunk 1]
```diff
@@ -3576,11 +3599,14 @@ public function postProcess() {
...
    * Set limit.
    *
-   * @param int $rowCount
+   * @param int|null $rowCount
    *
    * @return array
    */
-  public function limit($rowCount = self::ROW_COUNT_LIMIT) {
+  public function limit($rowCount = NULL) {
+    if (CRM_Utils_System::isNull($rowCount)) {
```
Comment (demeritcowboy) [inline CRM/Report/Form.php:3607]: I see isNull isn't deprecated yet but it does unexpected things sometimes. Was this copy/paste or is there a reason to use it?

Comment (seamuslee001) [reply CRM/Report/Form.php:3607]: I was wanting to get the function if $rowCount was NULL rather than say 0 had been passed into the function

Comment (demeritcowboy) [reply CRM/Report/Form.php:3607]: In php 7.4 there's a nifty `$rowCount ??= $this->getRowCount();`. For php 7 in general could do `$rowCount = $rowcount ?? $this->getRowCount();` or there's regular `is_null()`.

**Comment by seamuslee001** [comment]
@demeritcowboy I believe I have updated this to what you wanted now

**Pull request merged**